data/hai/unfettered side missions.txt

```diff
@@ -0,0 +1,1660 @@
+				die
+			label steal
+			`	You manage to enter the ship discreetly. Some people will always notice, but you try to pose as a spaceport worker there to assist with the repairs, using equipment you find lying around. You should be able to go pretty much anywhere on the ship. You don't have any idea which part of the ship is actually damaged though, so your cover could easily be blown. There is no going back now, except maybe if you choose to provoke the warlord into a fight to get the item.`
+			choice
+				`	(Search the lower decks.)`
+				`	(Search close to the weapons stations.)`
+					goto weapons
+				`	(Search the bridge.)`
+					goto bridge
+				`	(Look near the engines.)`
+					goto engine
+				`	(Check among the cargo.)`
+					goto cargo
+				`	(Search the upper decks.)`
+				`	(Leave the ship before you get caught, and provoke the warlord.)`
+					goto challenge
+			`	You notice a lot of security stations around, they obviously care a lot about their warlord's security.`
+				goto caught
+			label weapons
+			`	This part of the ship is mostly full of torpedoes - apparently the warlord loves them. But there are not many people here, since this part of the ship seems to have suffered no damage. The torpedoes will probably be restocked soon, so you leave before someone comes in and notices you.`
+				goto found
+			label bridge
+			`	This bridge is very luxurious, you see that this warlord likes comfort, for herself at least. But you cannot find anything that looks unique at all.`
+				goto caught
+			label cargo
+			`	There are a lot of exotic things here, most of which are probably not legal at all. You better choose what to take with you, and fast! You also notice the presence of gold bars. There's no way you could carry all of them back with you discreetly, but you can drop them off the ship and then go grab them once you are out, hoping nobody will notice. Looking through a porthole you don't see anyone around at the moment. Conveniently, there is even some sand on the ground there, so the bars should sink in, partially hidden under it.`
+			branch knowsbars
+				has "Unfettered: Betrayal For The Bars: declined"
+			`	It may not be what you're looking for, but these bars are probably worth a small fortune, and you reckon you could get away with a few tons of them. They have special markings on them and are probably worth more than the value of the gold alone! You can always take out the pirate lord's ship afterward and claim them as your own, as they are more heavy and take more space than any of the other items which could interest the warlord.`
```

Comment:
Just casually walk away with a couple tons of gold. This feels like minecraft :)

Reply:
I think I make the player throw them out the window and get them later

Reply:
It's mostly fine, just weird

Reply:
I specified its using an antigrav unit that you do this, and the help of a crew member